data-pipeline/src/data_pipeline/data_types/cnv_coverage.py

```diff
@@ -3,7 +3,7 @@
 
 
 def prepare_cnv_track_callable_coverage(coverage_path):
-    coverage = hl.import_table(coverage_path, force_bgz=True)
+    coverage = hl.import_table(coverage_path, force_bgz=True, types={'xpos': hl.tfloat64, 'percent_callable': hl.tfloat64})
```

Comment:
There are a couple of files in this commit that are deleted by the end of the PR, they should be removed.